plugins/workspace-backpack/src/backpack.js

```diff
@@ -440,14 +440,18 @@ export class Backpack extends Blockly.DragTarget {
       this.addBlock(/** @type {!Blockly.BlockSvg} */ (dragElement));
     }
   }
+
   /**
-   * Converts the provided block into a cleaned XML string.
+   * Converts the provided block into a JSON string.
    * @param {!Blockly.Block} block The block to convert.
-   * @returns {string} The cleaned XML string.
-   * @private
+   * @returns {string} The JSON object as a string
    */
-  blockToCleanXmlString_(block) {
-    return cleanBlockXML(Blockly.Xml.blockToDom(block));
+  blockToJSONString_(block) {
```

Comment:
nit: The google javascript styleguide suggests `blockToJsonString`

https://google.github.io/styleguide/jsguide.html#naming

You'll also want to mark this `@private` unless you're calling it from a different class!